src/UI/ManualImport/Album/SelectAlbumLayout.js

```diff
@@ -0,0 +1,71 @@
+var _ = require('underscore');
+var vent = require('vent');
+var Marionette = require('marionette');
+var Backgrid = require('backgrid');
+var LoadingView = require('../../Shared/LoadingView');
+var AlbumCollection = require('../../Artist/AlbumCollection');
+var SelectAlbumRow = require('./SelectAlbumRow');
+
+module.exports = Marionette.Layout.extend({
+    template  : 'ManualImport/Album/SelectAlbumLayoutTemplate',
+
+    regions : {
+        album : '.x-album'
+    },
+
+    ui : {
+        filter : '.x-filter'
+    },
+
+    columns : [
+        {
+            name      : 'title',
+            label     : 'Title',
+            cell      : 'String',
+            sortValue : 'title' //TODO Change to Sort Title
```

Comment:
Does this need to be done before merge?

Reply:
This may be best for separate PR, will require adding field to album db table, model, and resource.